src/storage/src/mem_table.rs

```diff
@@ -434,6 +436,7 @@ pub struct MemtableLocalStateStore<S: StateStoreWrite + StateStoreRead> {
     table_id: TableId,
     op_consistency_level: OpConsistencyLevel,
     table_option: TableOption,
+    vnodes: Option<Arc<Bitmap>>,
```

Comment:
For simplicity I think it's unnecessary to store the vnode in this trivial local state store.

Reply:
If we don't store vnodes in this local state store, I think the assertion in state table to check whether the state table and state store vnode bitmap matches will fail.